eng/common/scripts/modules/Package-Properties/Package-Properties.psm1

```diff
@@ -0,0 +1,172 @@
+# Helper functions for retireving useful information from azure-sdk-for-* repo
+# Example Use : Import-Module .\eng\common\scripts\modules\Package-Properties\Package-Properties.psm1 -ArgumentList 'JavaScript','C:\Git\azure-sdk-for-js'
```

Comment:
Why put this in its own directory? If we start using more modules we can put them all in the modules folder and we can import them all with a single import command `Import-Module .\eng\common\scripts\modules`, that is assuming we don't need to pass params. We probably should add any params to the exported functions so we don't need to depend on any global stored state.